snippets/native-platforms/ios-swift/use.md

```diff
@@ -2,7 +2,7 @@
 let lock = MyApplication.sharedInstance.lock
 let controller = lock.newLockViewController()
 controller.closable = true
-controller.onAuthenticationBlock = {(profile:A0UserProfile!, token:A0Token!) -> () in
+controller.onAuthenticationBlock = { (profile, token) -> () in
```

Comment:
For the sake of removing unnecessary code I think you can remove the `-> ()` :smile: